From e906a4c6d0fd64c60cdd39055837f9ef3ee6cef5 Mon Sep 17 00:00:00 2001
From: Michal Suchanek <msuchanek@suse.de>
Date: Mon, 25 Feb 2019 10:03:46 +0100
Subject: [PATCH] Revert "mm, devm_memremap_pages: mark devm_memremap_pages()
 EXPORT_SYMBOL_GPL"

References: bnc#1012382
Patch-mainline: no, kABI

This reverts commit a93d56de4533839f6cdb5865bf91d01f02eb10e9.

Signed-off-by: Michal Suchanek <msuchanek@suse.de>
---
 kernel/memremap.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/memremap.c b/kernel/memremap.c
index 1be42f9b3e00..887fbab7d8d3 100644
--- a/kernel/memremap.c
+++ b/kernel/memremap.c
@@ -199,5 +199,5 @@ void *devm_memremap_pages(struct device *dev, struct resource *res)
 	devres_add(dev, page_map);
 	return __va(res->start);
 }
-EXPORT_SYMBOL_GPL(devm_memremap_pages);
+EXPORT_SYMBOL(devm_memremap_pages);
 #endif /* CONFIG_ZONE_DEVICE */
-- 
2.20.1

